protoc-gen-bq-json-schema/genjson/config.go

```diff
@@ -43,4 +43,10 @@ func (c *Config) AddToFlagSet(flags *flag.FlagSet) {
 		false,
 		"Set the mode of a field to REQUIRED if the field is defined with REQUIRED behavior in proto.",
 	)
+	flags.BoolVar(
+		&c.SchemaOptions.UseProtoCommentsAsDescription,
+		"proto_comments_as_description",
+		true,
```

Comment:
Maybe set this to false and only set it to true for the comments example? Will that be possible? 🤔

Reply:
It's not possible to set it individually since those are created with `buf generate` when building.
I think it makes sense to have it enabled by default, and then make it possible to disable.